src/meetups/2020-11-11.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: meetups
+location: Remote
+started_at: 2020-11-11 17:30:00
```

Comment:
This time is in UTC rather than the current time of UK. Now that we've changed the clocks this can be `2020-11-11 18:30:00`. `ended_at` can bean hour later too.

Reply:
As you can see my brain is not fully firing on all cylinders 😅 Thank you for the proofread! Changes made.